source/assets/scripts/recipe-detail.js

```diff
@@ -91,9 +91,9 @@ function bindFoodieButton() {
 
 function goBack(){
   const btn = document.getElementById('white-arrow-p');
-  let index = document.referrer.lastIndexOf('/');
-  let str = document.referrer.substring(index + 1);
-  console.log(str);
+  // let index = document.referrer.lastIndexOf('/');
+  // let str = document.referrer.substring(index + 1);
+  // console.log(str);
 
   btn.addEventListener('click', () => window.history.back());
```

Comment:
For a saved recipe, after I entered the recipe from dashboard, clicked the saved button (cancel favorite), and go back to dashboard, the recipe is still in the saved recipe list. That's why I need to check whether str is index.html.